### ISSUE TITLE
Add Colgate Total Toothpaste Products

### ISSUE THREAD

**Issue by KAUTH** [ProductCrawler/product_crawler#44]
We need some people to add the following products: 

- [ ] **Colgate Total Advanced Fresh + Whitening Gel**
- [x] **Colgate Total Advanced Whitening Toothpaste**
- [ ] **Colgate Total Advanced Whitening Gel**
- [x] **Colgate Total Clean Mint Toothpaste**
- [ ] **Colgate Total Daily Repair Toothpaste**
- [x] **Colgate Total Fresh Mint StripeTM Gel Toothpaste**
- [ ] **Colgate Total Whitening Toothpaste**
- [x] **Colgate Total Whitening Gel**

(these are the product names) to the project.

These products can be categorized into the **Product Type** of _Hygiene_.
Here is a helpful link:
http://www.colgatetotal.com/health-benefits/toothpaste-ingredients

Read the **README.md** and **CONTRIBUTING.md** before making a PR and contributing.

Thanks for taking the time to contribute to our project, we appreciate your help! 🌟

**Comment by vrishank97** [comment]
Can I work on this?

**Comment by KAUTH** [comment]
@vrishank97 yes of course. Thanks for the contribution!! Will you make PRs for all the products or just for some of them?

**Comment by vrishank97** [comment]
I'll make a PR for -
Colgate Total Advanced Fresh + Whitening Gel
Colgate Total Advanced Whitening Toothpaste
Colgate Total Advanced Whitening Gel

**Comment by KAUTH** [comment]
@vrishank97 that is great!! Thanks

**Comment by manaswinidas** [comment]
Can I make a PR for the remaining products

**Comment by KAUTH** [comment]
@manaswinidas yes, thank you very much!!!

**Comment by manaswinidas** [comment]
Am I supposed to create a single folder for Colgate products? I can see a Colgate Total Advanced Deep Clean Toothpaste folder already existing. Am I supposed to make changes there?

**Comment by KAUTH** [comment]
@manaswinidas make different PRs therefore different folders for each Colgate product

**Comment by manaswinidas** [comment]
Thank you!

**Comment by manaswinidas** [comment]
@KAUTH What am I supposed to do when ingredients aren't given. I am attaching a picture below. Can I attach this image or any changes need to be done?
![crop](https://user-images.githubusercontent.com/22885912/31334268-3a1f9ee8-ad0b-11e7-9d0a-9425b286ea13.png)

**Comment by Iqlaas** [comment]
To those who have contributed, kindly tick your products above so that there will no be duplicate PR.

**Comment by kondureal** [comment]
I'll do the product Colgate Total Advanced Whitening Toothpaste
